docs/operations/migration_policies.md

```diff
@@ -0,0 +1,161 @@
+# Migration Policies
```

Comment:
We need a disclaimer that LMPs are still Alpha

Reply:
Thanks, done!